packages/terra-tabs/src/Tabs.jsx

```diff
@@ -151,98 +85,43 @@ class Tabs extends React.Component {
       children,
       activeKey,
       defaultActiveKey,
-      responsiveTo,
       ...customProps
     } = this.props;
 
-    const theme = this.context;
-
-    // NOTE: Hardcoding variant to structural until discussions have resolved around if we want modular tabs.
-    const variant = 'structural';
-    const tabsClassNames = classNames(cx(
-      'tabs-container',
-      { 'tab-fill': tabFill },
-      variant,
-      theme.className,
-    ),
-    customProps.className);
-
-    let content = null;
-    let isIconOnly = false;
-    const clonedPanes = [];
-    React.Children.forEach(children, (child, index) => {
-      let isActive = false;
-      if (child.key === this.state.activeKey || child.key === activeKey) {
-        isActive = true;
-        content = child.props.children;
-      }
+    const commonTabItems = [];
 
-      if (child.props.isIconOnly) {
-        isIconOnly = true;
+    React.Children.forEach(children, child => {
+      let content;
+      let workspaceContent;
+      if (child.key === this.state.activeKey) {
+        content = React.Children.map(child.props.children, contentItem => (
+          React.cloneElement(contentItem)
+        ));
+        workspaceContent = <CommonTabContent>{content}</CommonTabContent>;
```

Comment:
can we rename this to more meaningful name  like tabscontent ?

Reply:
Updated in https://github.com/cerner/terra-framework/pull/1693/commits/72db1fd101ce8c28b1afa9da2f2acddce6df20f6